src/OpenRiaServices.Client/Test/Client.Test/Data/OperationTests.cs

```diff
@@ -439,6 +442,91 @@ public void Bug706066_CancelInCallback()
             EnqueueTestComplete();
         }
 
+
+
+        class ExceptionHandleSynchronizationContext : SynchronizationContext
```

Comment:
You may also want to override Post so that it does something similar to what Send currently does (store exception)

Maybe `base.Post(_ => this.Send(d, state), state);` so that the test would work even if we change the implementation to do Post.